one-ui/ui/src/util/search-service.tsx

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios'
+
+const getResultsByQuery = async (database: string, ctsQuery: string, count: number, urisOnly: boolean) => {
```

Comment:
Can this be reworked to accept a flow name and step number? I'd be happy to make the DS endpoint to implement this, as this seems like too much implementation detail in the GUI.

Reply:
@rjrudin - We can definitely create an endpoint later which accepts only flow name and step number and returns the documents but in this case the mapping artifact is not part of any particular flow unless its added to it, as per the new one-ui design. So we might need to think about it in detail in another story.